server/main.go

```diff
@@ -115,6 +115,7 @@ func (b *Klein) create(w http.ResponseWriter, r *http.Request) {
 
 	// set an alias
 	alias := r.FormValue("alias")
+	overwrite := r.FormValue("overwrite")
```

Comment:
```suggestion
	overwrite := r.FormValue("overwrite") == "true"
```
quick way to turn this into a bool from the get go.